nautobot/core/tests/test_views.py

```diff
@@ -338,6 +338,11 @@ def query_and_parse_metrics(self):
         page_content = response.content.decode(response.charset)
         return text_string_to_metric_families(page_content)
 
+    @override_settings(METRICS_AUTHENTICATED=True)
```

Comment:
Given the way the `urls.py` is implemented, does overriding this at this point actually change the view registration retroactively? I'd kind of expect not.